scripts/install-worker.sh

```diff
@@ -74,6 +74,8 @@ sudo yum install -y \
 # Remove any old kernel versions. `--count=1` here means "only leave 1 kernel version installed"
 sudo package-cleanup --oldkernels --count=1 -y
 
+sudo yum versionlock kernel
```

Comment:
Can you move the installation of the versionlock plugin to `upgrade_kernel.sh` and then do the versionlock there, just to have everything in one place?